homeassistant/components/lcn/config_flow.py

```diff
@@ -70,7 +71,7 @@ class LcnFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    async def async_step_import(self, data: ConfigType) -> FlowResult:
+    async def async_step_import(self, data: dict[str, Any]) -> FlowResult:
```

Comment:
Please replace the remaining use of `ConfigType` in the module too.

Reply:
Done

Reply:
There's still one left on line 28.

Reply:
Yes, I saw that one but thought it might be a legit use of configtype.  Second look: I think you are right.  Done.